frontend/src/APIClients/queries/CommentQueries.ts

```diff
@@ -0,0 +1,46 @@
+import { gql } from "@apollo/client";
+import { QueryInformation } from "./StoryQueries";
+
+const COMMENT_FIELDS = `
+    id
+    commentIndex
+    content
+    storyTranslationContentId
+    `;
+
+export type CommentResponse = {
+  id: number;
+  userId: number;
+  commentIndex: number;
+  time: string;
+  resolved: boolean;
+  content: string;
+};
+
+export const buildCommentsQuery = (
+  storyTranslationId: number,
+  resolved: boolean | null,
+): QueryInformation => {
+  let queryString = gql`
```

Comment:
nitpick: May have asked this before, so sorry if I did. does it make more sense to flip the order of these? 
```typescript
if (resolved == null) { }
else {}
```

that reads better to me, but very subjective thing

Reply:
Im in favor of the current code as it reads "by default, only pass in storyTranslationId. If resolved is defined, pass it in". 
I'll let lynn break the tie! ball is in your court.

Reply:
+1 to keep it as is :)